buildpack.go

```diff
@@ -29,6 +30,11 @@ func (bp Buildpack) noAPI() Buildpack {
 	return bp
 }
 
+func (bp Buildpack) withHomepage(bpInfo BuildpackInfo) Buildpack {
```

Comment:
```suggestion
func (bp Buildpack) setHomepage(bpInfo BuildpackInfo) Buildpack {
```

Reply:
I kind of like `withHomepage` ...it's more clear that the `bp` of the calling fucntion isn't actually being modified.

Reply:
@natalieparellano, I agree. I missed this part.
Resolving this conversation.